utils/config/daos_server.yml

```diff
@@ -448,6 +448,8 @@
 #
 #  env_vars:
 #      - CRT_TIMEOUT=100
+#      - FI_OFI_RXM_USE_SRX=1 # set it to 0 if network card did not support
+#                               shared receive context, eg intel E810-C.
```

Comment:
you need a 2nd comment character here or unmarshalling will cause a test fail...
did not --> does not